### ISSUE TITLE
desserts is REALLY big

### ISSUE THREAD

**Issue by roymichaeldonald** [community-table-market/addison-county#22]
Not sure this is a bug or a feature request but the huge DESSERTS link on the restaurant page is odd

![big desserts](https://user-images.githubusercontent.com/70439642/91638881-8dd95c80-e9e0-11ea-970c-a87d807f3276.jpg)

**Comment by justindoran** [comment]
Yeah, it is the menu title, but it's pretty obnoxious. I've reduced the size of that and made it font-variant smallcaps instead of all caps﻿. In the next push (available Aug 30/31), I'll touch base here again and you can let me know what you think

**Comment by loretta-t** [comment]
Adding to this thread. In addition to the size, there are some alignment issues on mobile. See attached. 

![IMG-3709](https://user-images.githubusercontent.com/21224411/91649015-efc7af80-ea3c-11ea-8f2a-d99d565a82f7.PNG)
![IMG-3710](https://user-images.githubusercontent.com/21224411/91649016-f2c2a000-ea3c-11ea-8205-31c4524b5025.PNG)
![IMG-3711](https://user-images.githubusercontent.com/21224411/91649022-f6eebd80-ea3c-11ea-9de3-8d7d6233642e.PNG)

**Comment by loretta-t** [comment]
Further to this comment, the menu titles are underlined (typically indicating it's clickable) though they aren't linked.  Whereas the restaurant titles on the "Participating Restaurants" are not underlined, though they are clickable.
